Makefile

```diff
@@ -28,6 +28,8 @@ install-tools:
 	@go get -u golang.org/x/tools/cmd/goimports
 	@echo ">> fetching dep"
 	@go get -u github.com/golang/dep/cmd/dep
+	@go get -u github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
```

Comment:
you don't need these. These packages are marked as required in kedge `Gopkg.toml`

Reply:
this means that any `make dep` will fetch them

Reply:
ahh, I didn't run `dep ensure` when I first started working on the project it seems (or my GOBIN wasn't sourced). will remove